File: P16_u3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p16_u3;
import java.util.Scanner;
/**
 *
 * @author richy
 */
public class P16_u3 {

    public static void main(String[] args) {
        Scanner sc= new Scanner (System.in);
        int c=1, hora, pago, sueldo;
        while (c<=20)
        {
            System.out.println("Ingrese la cantidad de horas trabajadas: ");
            hora= sc.nextInt();
            System.out.println("Ingrese el pago de las horas: ");
            pago= sc.nextInt();
            sueldo= hora * pago;        
            System.out.println("Su sueldo semanal es: " + sueldo);
            c++;
        }
    }
}

File: p16_u3.cs
using System;

namespace p16_u3
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int i = 1, hora, pago, sueldo;
            while (i <= 20)
            {
                Console.WriteLine("Ingrese la cantidad de horas trabajadas: ");
                hora = Convert.ToInt32(Console.ReadLine());
                Console.WriteLine("Ingrese el pago de las horas: ");
                pago = Convert.ToInt32(Console.ReadLine());
                sueldo = hora * pago;
                Console.WriteLine("Su sueldo semanal es: " + sueldo);
                i++;
            }
        }
    }
}